identity-service/1.5/install/k8s/index.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Install Identity Service into Kubernetes Cluster
+---
+
+The Identity Service can be deployed into a new or existing Kubernetes cluster.
+
+## Prerequisites
+
+-   A Kubernetes cluster
+-   Helm and Tiller configured in the cluster
```

Comment:
I would not refer to Tiller anymore since we are using `helm 3` that is not installing it.